installers/aws-docker/install/env.sh

```diff
@@ -64,10 +64,14 @@ MONGO_HOST_PORT=$HOST_DNS_NAME:27017
 LEGEND_ENGINE_PORT=6060
 LEGEND_ENGINE_URL=http://$HOST_DNS_NAME:$LEGEND_ENGINE_PORT
 
-# LEGEND_SDLC
+# SDLC
 LEGEND_SDLC_PORT=7070
 LEGEND_SDLC_URL=http://$HOST_DNS_NAME:$LEGEND_SDLC_PORT
 
+# Metadata
+LEGEND_METADATA_PORT=8080
```

Comment:
Port conflict. Same default port as the SDLC server

Reply:
my bad 😢